config/routes.rb

```diff
@@ -5,5 +5,11 @@
     resources :friendships
   end
 
+  resources :projects do
+    resources :project_users
+  end
+
+  post 'project/:id/add' => 'projects#add', as: :project_add
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.